ami-recycle-bin/example-pattern.json

```diff
@@ -0,0 +1,55 @@
+{
+    "title": "Deregister expired AMIs and its snapshot with AWS Lambda and retain them in Amazon EC2 Recycle Bin",
```

Comment:
The title can have at most 75 characters, please reword

Reply:
> The title can have at most 75 characters, please reword

Updated title to 75 characters **AMI de-registration with AWS Lambda and retention in Amazon EC2 Recycle Bin**